src/tools/miri/src/diagnostics.rs

```diff
@@ -395,6 +395,7 @@ pub fn report_msg<'tcx>(
 ) {
     let span = stacktrace.first().map_or(DUMMY_SP, |fi| fi.span);
     let sess = machine.tcx.sess;
+    let title = title.to_string();
```

Comment:
Might as well change the type of `title` to `String` then. There shouldn't be many call sites.

Reply:
Done.